src/api/EscargotPublic.h

```diff
@@ -794,7 +794,12 @@ class ESCARGOT_EXPORT StringRef : public PointerValueRef {
 
     StringRef* substring(size_t from, size_t to);
 
-    std::string toStdUTF8String();
+    enum WriteOptions {
+        NO_OPTIONS = 0,
+        REPLACE_INVALID_UTF8 = 8
```

Comment:
If you need only `REPLACE_INVALID_UTF8` option, how about representing this option simply by boolean type instead of additional enum value for each parameter?

Reply:
Now I need only REPLACE_INVALID_UTF8 option. But v8 support more option that might be needed in future. Using a boolean would require an api change in future to support other like NO_NULL_TERMINATION or PRESERVE_ONE_BYTE_NULL. That is why I have used enum instead of boolean.